<launch>
    <!-- Configuration file argument -->
    <arg name="config" default="$(find height_mapping)/config/params.yaml" />

    <node name="height_mapping" pkg="height_mapping" type="height_mapping_node" output="screen">
        <rosparam file="$(arg config)" command="load" subst_value="true" />
    </node>

</launch>